### ISSUE TITLE
It shouldn't update redist-like packages

### ISSUE THREAD

**Issue by thany** [NexovaDev/UpdateHub#388]
I'm getting a lot of updates for C++ Redistributables, .NET SDK, that sort of things. First of all, they aren't apps. But more importantly, updating them can be dangerous, it can make programs stop functioning properly. When installing something that requires any of these packages, the installer of such program makes sure the correct one is installed.

Typically you should never touch them.

**Comment by LogicDaemon** [comment]
I disagree, in the days before Windows 7 I used to update C/C++ runtime libs manually; since then Windows includes them in Windows Update.
This has rarely caused problems.
All such updates are binary compatible.

At least MS's own dlls are installed with different filenames when compatibility is broken.

Otherwise the same exe wouldn't run on all systems from NT4.0 to Windows 11 (and currently you can run almost anything from NT4.0).

Linux is different: both `glibc` and the kernel can be binary incompatible between versions, so the whole system has to be recompiled to work with the new environment. But since almost all packages have source available, this isn't a big problem.
